scripts/results/arrange-data.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2016-2019, Regents of the University of Arizona.
+# Author:Wenkai Zheng<wenkaizheng@email.arizona.edu>
+# DESCRIPTION:
+# ------------
+# This file is used for arranging data and print out plot
+# firstly sort data into a map and sort data in  each session
+# change map to 2d array and then sort 2d array by every first element in each sub array.
+# print out start time of earliest and lastest time (2d array[0][0] and 2d array[-1][0] 
+# draw plot with specific time range 
+# if the start time of one session is in the range, we write all data in this session to a file
+# send file to drawing script and after finishing drawing, we delete this file.

```

Comment:
This is not good.
Few comments:
1. Finish sentences with a period.
2. Start sentences with capital letter.
3. You do not need to talk about implementation details. Just tell the reader what this script does. What the input is and what output they should expect.
4. Try not to use `we`.